utils/interfaces/_library/core.py

```diff
@@ -266,13 +266,23 @@ def add_appsec_reported_header(self, request, header_name):
     def add_traces_validation(self, validator, success_by_default=False):
         self.validate(validator=validator, success_by_default=success_by_default, path_filters=r"/v0\.[1-9]+/traces")
 
-    def validate_spans(self, request=None, validator=None, success_by_default=False):
+    def validate_spans(self, request=None, validator=None, success_by_default=False, validate_all_spans=False):
```

Comment:
Can you keep this function as is, to keep a consistent API. See above for the need "at least one span is valid"